scss/5-pages/_report.scss

```diff
@@ -0,0 +1,80 @@
+
+.site-header {
+	color:#4d4d4d;
+	margin-top: 5rem;
+	margin-bottom: 5rem;
+
+    h1 {
+        font-size: 60pt;
+        margin-top: 60pt;
+        font-weight: bold;
+    }
+
+    h2{
+    	margin-top:0px;
+    }
+
+    margin-bottom: 2em;
+}
+
+
+.container.report {
+	color:#4d4d4d;
+	margin-bottom: 45pt;
```

Comment:
Why pt ?
pt is **only for print** - see https://css-tricks.com/css-font-size/
If you want to use px, then use the *rem* mixin, otherwise use rem :)